jobs/service-fabrik-broker/templates/config/eventlog-config-external.yml.erb

```diff
@@ -97,7 +97,7 @@ defaults:
     - backup
 "/api/v1/backups/:backup_guid":
   GET:
-    enabled: true
+    enabled: false
```

Comment:
If these properties are frequently changing, we should make them part of the spec, In future then we can avoid boshrelease changes. If we do not want to pass values from manifest, we can put default values in spec for now.

Reply:
@subhankarc @kamath-prasad  These are somethings which will not change frequently. Whenever we want to tweak them they can go in as part of the release. If event config is to be driven via config then its a really big list and there is no need to change them on the fly and it can always wait till next release. So such configurations i recommend not to be put in as part of bosh spec and let us manage it within broker & update time to time whenever required.

Reply:
I don't see a problem if we change the default value in spec instead of in erb file. I guess that way, maintaining it is easier, also if you need to turn it off, just the manifest changes suffice.

Reply:
Yes , makes sense . I do not see this property getting changed on the fly as it is not critical.

Hence reverting the changes . please review and merge.

Reply:
@subhankarc Agree to your statements. The only issue with exposing this event config via spec is that its a huge list & we must then manage this spec list and also code the same in erb file to pick right value from spec. Changing the value in spec file or erb file is equivalent. Unless we dont change these often and dont need them to be done outside of release can keep them out of spec file. This is definitely debatable on both sides. :-)
Nevertheless if we want to parameterise then we must parameterise all events and not just the two events that are being turned off now as it leads the code in an inconsistent state. So I recommend the next time there is a CR on this item, we revisit how we address this overall.

Reply:
👍